includes/cub.h

```diff
@@ -213,6 +234,7 @@ char				*set_fps_string(char *str1, t_cub *cub, int mode);
 int update_display(t_cub *cub);
 int	key_up(int keycode, t_cub *cub);
 int	key_down(int keycode, t_cub *cub);
+int	exit_game(int keycode, t_cub *cub);
```

Comment:
Did you really use this function in the end? I don't see it

Reply:
Nope, it's a remnant from the old_days 👵🏻 , now that we have our friend graceful_exit 👑. Removing it